### ISSUE TITLE
Cannot create accounts

### ISSUE THREAD

**Issue by GabrielLeonte** [bitshares/bitshares-ui#1715]
Failed to create account: somename -  Uncaptured Faucet-API Exception Error loading Base58 object
Some help ?

**Comment by startailcoon** [comment]
Hi,

Did you use the Bitshares Reference wallet to sign up? 
It's located on wallet.bitshares.eu

**Comment by GabrielLeonte** [comment]
No. I m using a custom server with bitshare-ui and a custom chain

**Comment by GabrielLeonte** [comment]
I must install this to allow users to register ?  http://docs.bitshares.org/testnet/7-faucet.html
